astro/src/content/quickstarts/quickstart-android-java-native.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Android Java
+title: Android Java - AppAuth
```

Comment:
Because both the AppAuth quickstart and FusionAuth SDK quickstart is live at the same time it just makes sense to specify it more clearly especially as its used in the overview of quickstarts.